instances/tissueSampleType/nerve.jsonld

```diff
@@ -0,0 +1,14 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/tissueSampleType/nerve",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/TissueSampleType",
+  "definition": "A nerve sample (i.e. a whole nerve or a part of a nerve) from a living or deceased multicellular organism body.",
```

Comment:
suggestion "... from a neuron of a ..."

Reply:
I think we have different definitions for a nerve...

Merriam-Webster defines a nerve as "any of the filamentous bands of nervous tissue that connect parts of the nervous system with the other organs, conduct nerve impulses, and are made up of axons and dendrites together with protective and supportive structures". 

To me that's more than just a neuron, isn't it?

Reply:
no ... I had the same definition in mind (should have written neurons though)... but leave it as it is